kpi_dashboard/static/src/js/widget/meter_widget.js

```diff
@@ -17,7 +17,11 @@ odoo.define("kpi_dashboard.MeterWidget", function (require) {
         _getMeterOptions: function (values) {
             var size = Math.min(this.widget_size_x, this.widget_size_y - 40) - 10;
             return {
-                percent: values.value.value,
+                ...(values.value.value && {percent: values.value.value}),
+                ...(values.value.min && {min: values.value.min}),
+                ...(values.value.total && {total: values.value.total}),
```

Comment:
I don't understand why do you need so many parameters.

total is for max? used for value? In that case you can simplify all, isn't it?